pkg/server/auth/passthrough.go

```diff
@@ -35,5 +41,24 @@ func (pg *BearerTokenPassthroughPrincipalGetter) Principal(r *http.Request) (*Us
 		return nil, nil
 	}
 
-	return &UserPrincipal{Token: extractToken(token)}, nil
+	token = extractToken(token)
+
+	authv1.AddToScheme(pg.kubernetesClient.Scheme())
```

Comment:
We usually do all the `AddToScheme` at client creation. I don't know if `AddToScheme` is expensive etc, but would move if there just to keep that pattern, even though its harder to understand _why_ we're adding it all the way at the top.

Reply:
I only added it here for testing so I can pull the branch into WGE running in tilt. I will move it.